x/auth/client/tx_test.go

```diff
@@ -71,6 +71,66 @@ func TestReadTxFromFile(t *testing.T) {
 	require.Equal(t, txBuilder.GetTx().GetFee(), txBldr.GetTx().GetFee())
 }
 
+func TestReadTxsFromFile(t *testing.T) {
+	t.Parallel()
+
+	encodingConfig := moduletestutil.MakeTestEncodingConfig()
+	interfaceRegistry := encodingConfig.InterfaceRegistry
+	txConfig := encodingConfig.TxConfig
+
+	clientCtx := client.Context{}
+	clientCtx = clientCtx.WithInterfaceRegistry(interfaceRegistry)
+	clientCtx = clientCtx.WithTxConfig(txConfig)
+
+	// Set up 2 txs
+	txBuilders := make([]client.TxBuilder, 2)
+	// Set up tx 1
+	txBuilders[0] = txConfig.NewTxBuilder()
+	txBuilders[0].SetFeeAmount(sdk.Coins{sdk.NewInt64Coin("atom", 150)})
+	txBuilders[0].SetGasLimit(uint64(50000))
+	txBuilders[0].SetMemo("foomemo")
+	// Set up tx 2
+	txBuilders[1] = txConfig.NewTxBuilder()
+	txBuilders[1].SetFeeAmount(sdk.Coins{sdk.NewInt64Coin("atom", 200)})
+	txBuilders[1].SetGasLimit(uint64(60000))
+	txBuilders[1].SetMemo("foomemo2")
+
+	// Write txs to the file
+	encodedTx1, err := txConfig.TxJSONEncoder()(txBuilders[0].GetTx())
+	require.NoError(t, err)
+	encodedTx2, err := txConfig.TxJSONEncoder()(txBuilders[1].GetTx())
+	require.NoError(t, err)
+
+	tx1String := string(encodedTx1) + "\n"
+	tx2String := string(encodedTx2) + "\n"
+	jsonBatchTxsFile := testutil.WriteToNewTempFile(t, tx1String+tx2String)
+	jsonSingleTxFile := testutil.WriteToNewTempFile(t, tx1String)
+
+	// Read it back
+
+	// 2 txs case
+	decodedBatchTxs, err := authclient.ReadTxsFromFile(clientCtx, jsonBatchTxsFile.Name())
+	require.NoError(t, err)
+	require.Equal(t, len(decodedBatchTxs), 2)
+	for i, decodedTx := range decodedBatchTxs {
+		txBldr, err := txConfig.WrapTxBuilder(decodedTx)
+		require.NoError(t, err)
+		t.Log(txBuilders[i].GetTx())
+		t.Log(txBldr.GetTx())
+		require.Equal(t, txBuilders[i].GetTx().GetMemo(), txBldr.GetTx().GetMemo())
```

Comment:
Each line is quite involved and repetitive hence hard to follow moreover when we aren't sure which is the retrieved tx vs the one we want. Could we please extract this into a variable so
```go
wantTx := txBuilders[i].GetTx()
gotTx := txBldr.GetTx()

require.Equal(t, wantTx.GetMemo(), gotTx.GetMemo(), "mismatched memos")
require.Equal(t, wantTx.GetFee(), gotTx.GetFee(), "mismatched fees")
```